apps/meteor/app/livechat/server/lib/Livechat.js

```diff
@@ -773,18 +773,20 @@ export const Livechat = {
 		const options = {
 			data: postData,
 			...(secretToken !== '' && secretToken !== undefined && { headers }),
+			timeout: settings.get('Livechat_http_timeout'),
 		};
 		try {
 			return HTTP.post(settings.get('Livechat_webhookUrl'), options);
 		} catch (err) {
 			Livechat.webhookLogger.error({ msg: `Response error on ${11 - attempts} try ->`, err });
-			// try 10 times after 10 seconds each
-			attempts - 1 && Livechat.webhookLogger.warn('Will try again in 10 seconds ...');
+			// try 10 times after 20 seconds each
+			attempts - 1 &&
+				Livechat.webhookLogger.warn(`Will try again in ${(settings.get('Livechat_http_timeout') / 1000).toPrecision(2) * 4} seconds ...`);
 			setTimeout(
 				Meteor.bindEnvironment(function () {
 					Livechat.sendRequest(postData, callback, attempts - 1);
 				}),
-				10000,
+				settings.get('Livechat_http_timeout') * 4,
```

Comment:
If there aren't many issues with typing on this `sendRequest` method, could you please move this entire method to the `LivechatTyped.ts` file located in this same folder?